OnlineFlightSearchAPITestCases/AirportServicesTests/AirportServicesTestCases.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using OnlineFlightSearchAPI.FlightServices;
+using Unity;
+using Xunit;
+
+namespace OnlineFlightSearchAPITestCases
+{
+    public class AirportServicesTestCases
+    {
+
+        private IUnityContainer Initialize()
+        {
+            IUnityContainer container = new UnityContainer();
+            container.RegisterType<IAirportServices, AirportServices>();
+            return container;
+        }
+
+        [Theory]
+        [InlineData("BUD")]
+        [InlineData("IAD")]
+        public void AirportValidation_CheckIfAirportCodeIsValid_ReturnsTrue(string airportCode)
```

Comment:
Testing AirportValidation? or CheckIfAirportIsValid?
Applies to all tests. 
A typical test name should make it easier to understand the unit under test, testing conditions and expected behavior. Go through Art of unit testing book

Reply:
Yep renamed as per suggestion